tests/unit/components/materialize-button-test.js

```diff
@@ -40,7 +40,7 @@ test('text test', function(assert) {
     component.set('text', 'Button');
   });
 
-  assert.equal(this.$().html().trim(), 'Button');
+  assert.equal(this.$().text().trim(), 'Button');
```

Comment:
This test broke as a result of bumping up to ember 1.11. Looking at the button component, there's nothing that would be adding an empty HTML comment to the button contents, so perhaps it's something in the framework. 

````
not ok 48 PhantomJS 2.0 - MaterializeButtonComponent: text test
    ---
        actual: >
            <!---->Button
        expected: >
            Button
        Log: >
    ...
````